README.md

```diff
@@ -97,6 +97,38 @@ Before we introduce the quantification assumptions, we need to mention that the
 *note*: make sure that each operations is performed on an independent set of triples. That is, for example, a value update shall not be conflated with a movement, or a value addition shall not be conflated with an update. 
 
 
+# Naming conventions
+## Operations
+Looking at patterns of change likely to occur in the context of maintaining SKOS vocabularies or to find in a diffing context
+we've identified 5 change types and mapped them for easy referencing as follows:
+* Addition --> Added  
+* Deletion --> Deleted
+* Value update --> Updated
+* Movement (cross property) --> Changed
```

Comment:
I would accept this as a convention, but it might be a source of confusion for the readers, as Updates and Changes are syninimous.

Reply:
It would be nice to also elaborate on each of those explaining what each does and showing an example.